scripts/buildkite/tpp-mlir.yml

```diff
@@ -9,21 +9,21 @@ steps:
   - wait
 
   - label: "TPP-MLIR-gcc-rel"
-    command: "${SRUN} --partition=spr-all --time=0:30:00 -- \
+    command: "${SRUN} --partition=emr --time=0:30:00 -- \
```

Comment:
please rebase

Reply:
The main: https://github.com/plaidml/tpp-mlir/blob/main/scripts/buildkite/tpp-mlir.yml still points to `spr-all`, hence updated to `emr`.

Reply:
oh, we don't use buildkite any more, ignore those

Reply:
Rebase, remove this file.